indigo/bulk_creator.py

```diff
@@ -352,6 +404,18 @@ def create_work(self, row, idx):
                 else:
                     row.errors = str(e)
 
+    def create_work(self, row, idx):
+        # handle spreadsheet that still only uses 'principal'
+        row['stub'] = row.get('stub') if 'stub' in row else not row.get('principal')
```

Comment:
can this be handled by `clean`, like you do with commenced etc.?

Reply:
by the time `clean` gets a hold of it, `False` has already been assigned to 'stub' if it's missing from the actual spreadsheet, so I'm suggesting doing it during `sanitize_incoming` instead, when we still have access to the row data and can check if 'stub' is in the dict.

Reply:
`clean` has access to both `self.data` and `self.cleaned_data` -- so I think you could still check the original `data` and adjust as necessary.

Reply:
thanks!